starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,25 @@
 // Iteration 1
 // 1. Create an array of strings
-
+let myArray: Array<string> = [];
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+  function addTask(task:string){
+  myArray.push(task)
+  console.log("task has been added");
+  return myArray.length;
+}
 // 3. Create a function to list all tasks, it must show in the console de task.
-
+function listAllTasks(){
+ myArray.forEach((e) => {
+   console.log(e)
+ })
+};
 // 4. Create a function to delete a task, you must find the task inside the array and delete it.
```

Comment:
Teneis que distinguir entre lo que devuelve la funcion y lo que le meteis. En este caso solo recibe un parametro string, que es task. Lo que devuelve es el number, por ejemplo:

let deleteTask = function (task: string): number {
  var index = this.tasks.indexOf(task);
  this.tasks.splice(index, 1);

  console.log("=========== NEW DELETION ===========");
  console.log(`Task "${task}" removed from the list`);

  return this.tasks.length;
}